resources/views/user/edit.blade.php

```diff
@@ -52,9 +64,15 @@
         </div>
         {!!Html::script('js/crop.js')!!}
         <script>
+            function ConfirmDelete() {
+              var conf = confirm("Are you sure want delete avatar?");
```

Comment:
You can return simple
return  confirm("Are you sure want delete avatar?")

instead of 
var conf = confirm("Are you sure want delete avatar?");
 +                if(conf){ return true;
 +                } else { return false;
 +              }

Reply:
Ok

Reply:
It doesn't work with this simple confirm instead my function